views/Hangouts.tsx

```diff
@@ -1,20 +1,128 @@
 import React from 'react'
-import { View, Text, Image, StyleSheet, TextInput } from "react-native"
-import UserList from "../components/UserList"
+import { View, Text, Image, StyleSheet, Button } from 'react-native'
+import UserList from '../components/UserList'
+import { connect } from 'react-redux'
+import Modal from 'react-native-modal'
+import { postHangoutAccept } from '../src/actions/userActions'
 
-const Hangouts = props => {
+interface UserLimited {
+  first_name: string
+  email: string
+  profile_photo: string
+}
+
+interface Props {
+  receivedHangoutRequests: Array<UserLimited>
+  acceptRequest: Function
+  currentUserEmail: string
+}
+
+class Hangouts extends React.Component<Props> {
+  state = {
+    modalVisible: this.props.receivedHangoutRequests.length > 0
```

Comment:
if a user has received any hangout requests, the modal will render with a list of them